docs/fides/docs/development/development_tips.md

```diff
@@ -28,7 +32,7 @@ SQLAlchemy is an Object Relational Mapper, allowing us to avoid writing direct d
 Database tables are defined with model classes. Model files should live in `src/app/models/`. Individual model classes must inherit from our custom base class at `app.db.base_class.Base` to ensure uniformity within the database. Multiple models per file are encouraged so long as they fit the same logical delineation within the project. An example model declaration is added below. For a comprehensive guide see: <https://docs.sqlalchemy.org/en/14/orm/mapping_styles.html#declarative-mapping>
```

Comment:
> Model files should live in `src/app/models/

Traditional ctl and ops models are in different places, not here

Reply:
Good catch, I copy/pastad this from Ops at some point it seems! I'll clean this up and hopefully soon enough it'll only be in a single place